specs/regression.txt

```diff
@@ -1702,3 +1702,15 @@ bar \
 <h1>foo \</h1>
 <p>bar \</p>
 ````````````````````````````````
+
+ISSUE 779
+
+```````````````````````````````` example
+[third try]:
+-
+
+[third try]
+.
+<h2>[third try]:</h2>
+<p>[third try]</p>
```

Comment:
This paragraph should not be emitted, but only the `h2`. This must be solved before merging. Thanks!

Reply:
Sorry, I missed the last `[third try]`. It's a bit weird to see a duplicated paragraph after the `h2`, but on the other hand it's useful to test it provides a correct paragraph when it is not in an `h2`.